main/input_default.h

```diff
@@ -272,6 +271,8 @@ class InputDefault : public Input {
 	virtual void flush_accumulated_events();
 	virtual void set_use_accumulated_input(bool p_enable);
 
+	List<Ref<InputEvent> > accumulated_events;
```

Comment:
This is a hideous hack, please find another way.

Reply:
What is hacky about it, I don't understand? I don't edit the list, I  just read it.

Reply:
I think @reduz's point is about the original code, not your changes. It seems he'd rather have a totally different approach for detecting echo than a fix over the current one.